.github/workflows/ci.yml

```diff
@@ -47,7 +47,7 @@ jobs:
         run: |
           python -m pip install --upgrade pip setuptools wheel
           pip install astropy click cython numpy pydantic pyyaml regions scipy
-          pip install pytest pytest-astropy
+          pip install pytest pytest-astropy requests tqdm
```

Comment:
If you do not install your dependencies explicitly, you directly test if your setup.py lists the correct dependencies.

Instead, use `extras` and do something like `pip install '.[all]'`